transformers/uws/server.js

```diff
@@ -46,7 +46,12 @@ module.exports = function server() {
       });
     });
 
-    this.service.upgrade(soc._handle.fd, req.headers['sec-websocket-key'], soc.ssl ? soc.ssl._external : null);
+    var ticket = this.service.transfer(soc._handle.fd, soc.ssl ? soc.ssl._external : null);
+    soc.on('close', function(error) {
+      if (!error) {
```

Comment:
What I mean is: Can we get rid of this check? Does `socket.destroy()` fires a `close` event with a `true` argument?